cyberbattle/simulation/generate_network.py

```diff
@@ -276,7 +276,7 @@ def create_node_data_without_vulnerabilities(node_id: m.NodeID):
             graph.nodes[node].update({'data': node_data})
 
     # remove all the edges inherited from the network graph
-    graph.clear_edges()
+    # graph.clear_edges()
```

Comment:
Was this causing an issue with your environment?

Reply:
Ah good catch. I assumed that was from my code. Looks like my environment was claiming `clear_edges` was not defined for `DiGraph[str]`. Just a typing error locally but doesn't appear in runtime. I will remove this from my PR.

Reply:
Actually this fails CI with the same error... maybe clear_edges is no longer supported by nx or something? Not sure.

Reply:
Try to rerun `pip install -r requirement.txt`, it should install version 2.7.1 which has `clear_edges`
https://networkx.org/documentation/networkx-2.7/reference/classes/generated/networkx.DiGraph.clear_edges.html#networkx.DiGraph.clear_edges

If pyright still fails, try deleting the `typing` folder and recreating it by calling `createStubs.sh`

Reply:
Ah this was because it was defaulting to my global pyright rather than venv pyright